package.json

```diff
@@ -75,15 +81,20 @@
     "eslint": "^7.26.0",
     "eslint-config-standard": "^16.0.2",
     "eslint-config-vuetify": "^0.5.0",
+    "eslint-import-resolver-node": "^0.3.4",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-standard": "^5.0.0",
+    "eslint-plugin-promise": "^5.1.0",
```

Comment:
* `standard` is used in the `.eslint.rc`, is it somewhere else in the dependency tree?
* `promise` is not used in the `.eslint.rc`, is it used somewhere else?

Reply:
Not used. It was to fix a warning I found when I started migrating to V2 Berry... then went to V3 + PnP, finally settling on V3 with `node_modules`. Somewhere along the way I think this was not necessary any longer :shrug: removed